services/vpc_manager/src/main/java/com/futurewei/alcor/vpcmanager/config/DefaultValueConfig.java

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.vpcmanager.config;
+
+public class DefaultValueConfig {
+
+    public static Integer mtu = 1450;
```

Comment:
Let us make it upper case for constant, like MTU.

Reply:
Sure